selfdrive/car/subaru/carstate.py

```diff
@@ -77,8 +77,17 @@ def update(self, cp, cp_cam, cp_body):
       ret.cruiseState.nonAdaptive = cp_cam.vl["ES_DashStatus"]["Conventional_Cruise"] == 1
       ret.cruiseState.standstill = cp_cam.vl["ES_DashStatus"]["Cruise_State"] == 3
       ret.stockFcw = cp_cam.vl["ES_LKAS_State"]["LKAS_Alert"] == 2
-      self.es_lkas_msg = copy.copy(cp_cam.vl["ES_LKAS_State"])
-
+      ret.stockAeb = (cp_cam.vl["ES_LKAS_State"]["LKAS_Alert"] == 5) or \
```

Comment:
can we split this out into another PR?

Reply:
AEB signal is currently last unresolved item (see above discussion). I would prefer using ES_Brake Cruise_Brake_Active signal to trigger AEB passthru since Cruise_Brake_Active indicates actual braking vs using dash alerts that trigger before braking. What do you think?

Reply:
I will do separate stockAeb PR